requester/requester_test.go

```diff
@@ -133,3 +135,45 @@ func TestBody(t *testing.T) {
 		t.Errorf("Expected to work 10 times, found %v", count)
 	}
 }
+
+func TestOutput(t *testing.T) {
+	var count int64
+	handler := func(w http.ResponseWriter, r *http.Request) {
+		atomic.AddInt64(&count, int64(1))
+	}
+	server := httptest.NewServer(http.HandlerFunc(handler))
+	defer server.Close()
+
+	req, _ := http.NewRequest("GET", server.URL, nil)
+	w := &Work{
+		Request: req,
+		N:       20,
+		C:       2,
+		Output:  "csv",
+	}
+	stdOut := readStdout(w.Run)
+	if count != 20 {
+		t.Errorf("Expected to work 20 times, found %v", count)
```

Comment:
Let's use got/want in the assertions. It makes it easier to refactor them in the future. Also follow got/want style error messaging.

if got, want := count, 20; got != want {
t.Errorf("count = %v; want %v", got, want)
}